cmd/minikube/cmd/root.go

```diff
@@ -75,6 +75,19 @@ var RootCmd = &cobra.Command{
 			}
 		}
 
```

Comment:
We need to add this here instead L#129 because this is the place where all required directory get initialized irrespective to what command we provide.